modules/aws_k8s_base/tf_module/ingress_nginx.tf

```diff
@@ -14,7 +54,7 @@ resource "helm_release" "ingress-nginx" {
         podLabels : {
           "opta-ingress-healthcheck" : "yes"
         }
-        extraArgs : var.private_key != "" || var.expose_self_signed_ssl ? { default-ssl-certificate : "ingress-nginx/secret-tls" } : {}
+        extraArgs : {}
```

Comment:
should nginx also use the ACM certificate?
by setting this empty, it will use the _nginx_ self-signed certificate.
This is probably fine - I'm not sure if we would have to rotate it when it expires, of if they take care of it.

Reply:
Huh, we use the default self-signed cert nginx comes up with all the time for nlb -> nginx connection. Should verify the expration, but that's for another PR

Reply:
Excellent question